test/conformance/blue_green_test.go

```diff
@@ -37,7 +37,7 @@ import (
 )
 
 const (
-	concurrentRequests = 100
+	concurrentRequests = 48
```

Comment:
amazing - this reminds me of OpenGL and using texture sizes that are a power of 2

Reply:
Yeah, I thought about 64 first but that may be high.

Probably better to use truncation, now that I think more about it.